src/Console/ConsoleOutput.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Console;
+
+use Infection\Mutant\Exception\MsiCalculationException;
+use Infection\Mutant\MetricsCalculator;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Process\Process;
+
+final class ConsoleOutput
+{
+    const CI_FLAG_ERROR = 'The minimum required %s percentage should be %s%%, but actual is %s%%. Improve your tests!';
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    public function __construct(SymfonyStyle $io)
+    {
+        $this->io = $io;
+    }
+
+    public function logVerbosityDeprecationNotice(string $valueToUse)
+    {
+        $this->io->note('Numeric versions of log-verbosity have been deprecated, please use, ' . $valueToUse . ' to keep the same result');
+    }
+
+    public function logUnkownVerbosityOption(string $default)
+    {
+        $this->io->note('Running infection with an unknown log-verbosity option, falling back to ' . $default . ' option');
+    }
+
+    public function logInitialTestsDoNotPass(Process $initialTestSuitProcess, string $testFrameworkKey)
+    {
+        $lines = [
+            'Project tests must be in a passing state before running Infection.',
+            sprintf(
+                '%s reported an exit code of %d.',
+                ucfirst($testFrameworkKey),
```

Comment:
while we at it, could you please change it to `AbstractTestFrameworkAdapter::getName()`